packages/dropdowns/stories/examples/Autocomplete/Default.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useRef, useEffect } from 'react';
+import { Story } from '@storybook/react';
+import {
+  Dropdown,
+  Menu,
+  Autocomplete,
+  Field,
+  Label,
+  Item,
+  Message
+} from '@zendeskgarden/react-dropdowns';
+import { Col, Row } from '@zendeskgarden/react-grid';
+import debounce from 'lodash.debounce';
+import { VALIDATION } from '../../../src/utils/validation';
+
+const options = [
+  'Asparagus',
+  'Brussel sprouts',
+  'Cauliflower',
+  'Garlic',
+  'Jerusalem artichoke',
+  'Kale',
+  'Lettuce',
+  'Onion',
+  'Mushroom',
+  'Potato',
+  'Radish',
+  'Spinach',
+  'Tomato',
+  'Yam',
+  'Zucchini'
+];
+
+interface IStoryProps {
+  placement:
+    | 'auto'
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'end'
+    | 'end-top'
+    | 'end-bottom'
+    | 'start'
+    | 'start-top'
+    | 'start-bottom';
+  isAnimated: boolean;
+  isCompact: boolean;
+  isBare: boolean;
+  focusInset: boolean;
+  validation: VALIDATION;
+  showMessage: boolean;
+  disabled: boolean;
+}
+
+export const Default: Story<IStoryProps> = ({
+  isAnimated,
+  isCompact,
+  isBare,
+  focusInset,
+  placement,
```

Comment:
`placement` is not a valid prop for `Autocomplete`. See https://garden.zendesk.com/components/autocomplete#autocomplete